packages/styling/src/styles/getFocusStyle.ts

```diff
@@ -139,26 +139,26 @@ export function getFocusOutlineStyle(theme: ITheme, inset: number = 0, width: nu
  * @param borderColor - Color of the border.
  * @param borderRadius - Radius of the border.
  * @param borderType - Type of the border.
- * @param borderPosition - Position of the border relative to the input element (default to -1
+ * @param borderPosition - Position of the border relative to the input element (default to { top: -1, right: -1, bottom: -1, left: -1 }
  * as it's the most common border width of the input element)
  * @returns The style object.
  */
 export const getInputFocusStyle = (
   borderColor: string,
   borderRadius: string | number,
   borderType: 'border' | 'borderBottom' = 'border',
-  borderPosition: number = -1
+  borderPosition: { top: number; right: number; bottom: number; left: number } = { top: -1, right: -1, bottom: -1, left: -1 }
```

Comment:
Thanks for catching the mis-alignment issue here. Instead of this change here is what I purpose:
we should not set  left / right to 0 if there is no left / right border
```
export const getInputFocusStyle = (
  borderColor: string,
  borderRadius: string | number,
  borderType: 'border' | 'borderBottom' = 'border',
  borderPosition: number = -1
): IRawStyle => {
  const isBorderBottom = borderType === 'borderBottom';

  return {
    borderColor,
    selectors: {
      ':after': {
        pointerEvents: 'none',
        content: "''",
        position: 'absolute',
        left: isBorderBottom ? 0 : borderPosition,
        top: borderPosition,
        bottom: borderPosition,
        right: isBorderBottom ? 0 : borderPosition,
        [borderType]: `2px solid ${borderColor}`,
        borderRadius,
        width: isBorderBottom ? '100%' : undefined,
        selectors: {
          [HighContrastSelector]: {
            [borderType === 'border' ? 'borderColor' : 'borderBottomColor']: 'Highlight'
          }
        }
      }
    }
  };
};
```

Reply:
Yeap, seems a better approach. 
Thank you